src/lib/pageVisibilityTracker.js

```diff
@@ -23,8 +23,9 @@ class PageVisibilityTracker {
   addEventListener(func) {
     if (
       typeof document.addEventListener === 'undefined' ||
-      this.hidden === undefined
+      typeof this.hidden === 'undefined'
```

Comment:
saw this refactoring in other places as well. Is it really better than just checking for `=== undefined` ?

Reply:
`x === undefined` != `typeof x === 'undefined'`

The first one will throw an error be cause x is undefined but `typeof x === 'undefined'` will correctly return a boolean:
![image](https://user-images.githubusercontent.com/1494211/71625508-798e5700-2be8-11ea-9fe6-fd44f9801835.png)